### DISCUSSION TITLE
Error : You must pass a string or Handlebars AST to Handlebars.compile. You passed null

### DISCUSSION THREAD

**Discussion by tedtninety** [microsoft-search/pnp-modern-search#2796] [Q&A] [answered]
Hello,

I had noticed that the backend folks have updated PnP Search to now have new data sources to select from including, SharePoint CAML, OData and Static Data.

However all of a sudden,  it is putting out an error despite not  making any changes.

`You must pass a string or Handlebars AST to Handlebars.compile. You passed null `

I am using "Custom Template"

Any help would be greatly appreciated.

**Comment by wobba** [comment, marked as answer]
Some more details on repro would be useful.

**Reply by tedtninety** [reply]
My apologies, It seems to have been an error by a team member.
Cheers!
